cli/flox/doc/flox-install.md

```diff
@@ -7,20 +7,70 @@ header: "Flox User Manuals"
 
 # NAME
 
-flox-install - Install package(s) to environment.
+flox-install - install packages to an environment
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] install [ `<options>` ] `<package>` [ `<package>` ... ]
+```
+flox [<general options>] install
+     [-i <id>] <package>
+     [[-i <id>] <package>, ...]
+```
 
 # DESCRIPTION
 
-Install package(s) to environment.
+Install packages to an environment.
+
+Package installation is transactional.
+During an installation attempt the environment is built in order to validate
+that the environment isn't broken
+(for example, in rare cases packages may provide files that conflict).
+If the build fails, the attempt is discarded and the environment is unmodified.
+Once a build attempt succeeds the environment is atomically updated.
+
+If a requested package is already installed nothing is done.
```

Comment:
It's zero, which I think is you could take either way. On one hand the operation is idempotent i.e. you can request to install `ripgrep` multiple times and only get it once. That's good. On the other hand if made a typo and attempt to install `gcc12` instead of `gcc13` when `gcc12` is already installed, it won't fail.